src/app.re

```diff
@@ -1,10 +1,16 @@
 let text = ReasonReact.stringToElement;
 
+type operation = 
```

Comment:
yes, I should use `variant` instead of `string`, so `reason` will help me to catch the bugs!